doi-proxy/build.gradle

```diff
@@ -3,4 +3,9 @@ dependencies {
 
     runtimeOnly group: 'org.glassfish.jersey.core', name: 'jersey-common', version: project.ext.jerseyVersion
     implementation group: 'jakarta.ws.rs', name: 'jakarta.ws.rs-api', version: project.ext.jakartaVersion
+
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk-secretsmanager', version: project.ext.awsSdkVersion
+    implementation group: 'com.amazonaws', name: 'aws-java-sdk-sts', version: project.ext.awsSdkVersion
```

Comment:
Where are we using sts?

Reply:
we don't (anymore) is of couse replaced by nva-core/secrets